src/audio/lib/synthesis.c

```diff
@@ -1110,7 +1110,8 @@ Acmd* AudioSynth_ProcessSample(s32 noteIndex, NoteSampleState* sampleState, Note
                     case CODEC_REVERB:
                         reverbAddrSrc = (void*)0xFFFFFFFF;
                         if (gAudioCustomReverbFunction != NULL) {
-                            reverbAddrSrc = gAudioCustomReverbFunction(sample, numSamplesToLoadAdj, flags, noteIndex);
+                            reverbAddrSrc =
+                                (void*)gAudioCustomReverbFunction(sample, numSamplesToLoadAdj, flags, noteIndex);
```

Comment:
```suggestion
                            reverbAddrSrc = gAudioCustomReverbFunction(sample, numSamplesToLoadAdj, flags, noteIndex);
```
Instead, change 
`typedef s32 (*AudioCustomReverbFunction)(Sample*, s32, s8, s32);`
to
`typedef void* (*AudioCustomReverbFunction)(Sample*, s32, s8, s32);`